dynamo-zip-store/src/main/scala/dynamo_zip_store/DynamoZipStore.scala

```diff
@@ -0,0 +1,52 @@
+package dynamo_zip_store
+
+
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import fs2.Stream
+import software.amazon.awssdk.services.dynamodb.{DynamoDbAsyncClient, DynamoDbClient}
+import software.amazon.awssdk.services.dynamodb.model.{AttributeValue, GetItemRequest, PutItemRequest}
+import zip_partitioner.FileArchive
+
+import scala.jdk.CollectionConverters.MapHasAsJava
+
+object DynamoZipStore {
+
+  def uploadFilesToDynamoDB(client: DynamoDbAsyncClient, fileArchives: Stream[IO, FileArchive], tableName: String, keyColumnName: String, valueColumnName: String): Stream[IO, PutItemRequest] = {
+
+    fileArchives.foreach { fileArchive =>
+      val fileName =fileArchive.name
+      val fileBytes = fileArchive.compressedData
+
+      val item = Map(
+        keyColumnName -> AttributeValue.builder().s(fileName).build(),
```

Comment:
have a `private def asAttributeValue(v: String): AttributeValue` so you don't write this all the time